src/store/utils/generatePDF.ts

```diff
@@ -44,12 +44,25 @@ const chunkArray = (arr: any, n: any) => {
   return chunks;
 };
 
+
```

Comment:
Not a bad idea keeping this function local to the file for now 👍 .

The key is to disambiguate it from the other one 😂